src/ert/_c_wrappers/enkf/ert_workflow_list.py

```diff
@@ -183,20 +180,14 @@ def getContext(self) -> SubstitutionList:
     def __str__(self):
         return f"ErtWorkflowList with jobs: {self.getJobNames()}"
 
-    def addWorkflow(self, wf_name: str, wf_path: str):
-        self._add_workflow(wf_name, wf_path).setParent(self)
-
-    def addJob(self, job_name: str, job_path: str):
-        self._add_job(job_name, job_path)
-
     def hasJob(self, job_name: str) -> bool:
-        return self._has_job(job_name)
+        return job_name in self._workflow_jobs.keys
```

Comment:
should this be `return job_name in self._workflow_jobs.keys()`  ?  I guess keys is a function? :thinking:

Reply:
Sloppy work this. I've fixed the other one as well. Thanks!